lib/card/action.rb

```diff
@@ -44,7 +44,8 @@ def delete_old_actions
   class Action < ActiveRecord::Base
     belongs_to :card
     belongs_to :act,  :foreign_key=>:card_act_id, :inverse_of=>:actions 
-    has_many   :changes, :foreign_key=>:card_action_id, :inverse_of=>:action, :dependent=>:delete_all
```

Comment:
"changes" already means something to active record (Dirty), and rails started getting grumpy about that.